app/helpers/catalog_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module CatalogHelper
+  include Blacklight::CatalogHelperBehavior
+
+  ##
+  # Get the URL to a document's thumbnail image
+  # Overrides Blacklight behaviour
+  #
+  # @param [SolrDocument, Presenter] document
+  # @return [String]
+  def thumbnail_url document
+    representative_document = ::SolrDocument.find(document.thumbnail_id)
```

Comment:
I think the first two lines need an `if` check, or equivalent, against `thumbnail_id` being `nil` -- I just created a new Collection, without any works assigned to it and no representative thumbnail, and hit this condition on the front page

Reply:
Refactored this to show the default image if anything fails.